docs/development/chromium-development.md

```diff
@@ -18,7 +18,9 @@ See also [V8 Development](v8-development.md)
 It is possible to debug Chromium with Electron by passing
 `--build_debug_libcc` to the bootstrap script:
 
-    $ ./script/bootstrap.py -d --build_debug_libcc
+```bash
```

Comment:
Any particular reason some of these are `bash` and some are `sh`? We should probably just pick one. My vote would be `sh` because it's more generic but still always accurate.

Reply:
We have both `bash` and `sh` scattered all over the docs, I think if we were to fix them, we should open up another PR for it.

Reply:
+1 on preferring `sh` over `bash` in documentation

+1 on resolving `bash` v. `sh` inconsistencies in a followup. AFAIK we're not using any bash-only  extensions and any variant, e.g. `dash`, would also work